ibm/data_source_ibm_dl_gateways.go

```diff
@@ -31,6 +31,11 @@ func dataSourceIBMDLGateways() *schema.Resource {
 							Computed:    true,
 							Description: "Id of the data source gateways",
 						},
+						dlAuthenticationKey: {
+							Type:        schema.TypeString,
+							Computed:    true,
```

Comment:
same comment add sensitive : true

Reply:
This authentication key displays the CRN of the key rather than the key secret. So it is okay to show this information